materials/debugging-R.md

```diff
@@ -5,56 +5,61 @@ title: Debugging
 language: R
 ---
 
-## Setup
-
-**Send `debugging-example-unfixed.R` to students so they can follow along.**
-
-* Start with a new project including `debugging-example-unfixed.R`
-
-## Basic Strategy
-
-* How do we figure out what's wrong with a program
-    * Be a scientist
-	    * Hypothesis about what is wrong
-	    * Make one change that is expected to fix it
-		* Check if it worked
-		* **Do not change something without a reason**
-    * Read the error message
-    * Talk through the code (rubber duck programming)
-	* Observe what the code is doing
-
-
-## Observe
-
-*Run*
-
-* Read the error message and try to fix it
-* Where did code fail? Show traceback
-* Read the code
-* Look at the current state of the environment
-* This gives us a snapshot of what's going on in the code
-* Can't see inside the function
-
-
-## Run line by line
-
-* Run the code line by line checking each step
-* Trick once functions are involved
-
-
-## Print statements
+> Remember to
+>
+> * send `debugging-example-unfixed.R` to students so they can follow along.
+> * set up a new `Git` project that includes `debugging-example-unfixed.R`.
```

Comment:
@ethanwhite: Do you have a `debugging-example-unfixed.R` file? From the lecture material, I think the file is a modification of `decomposition-example.R`. If you don't have the file, I should be able to revise `decomposition-example.R` to line up with the lesson.